src/konfetti/vault/asynchronous.py

```diff
@@ -28,13 +28,24 @@ def cached_call(method):
     return inner
 
 
-@attr.s(slots=True)
+@attr.s
 class AsyncVaultBackend(BaseVaultBackend):
     is_async = True
 
     async def load(self, path, url, token):
+        from aiohttp import ClientConnectionError
+        from tenacity import retry_if_exception_type, stop_after_attempt, stop_after_delay, AsyncRetrying
+
+        retry = self.retry_object
```

Comment:
maybe we could have some method in the base class that will get the retry object?

e.g.

```
retry = self._get_retry(AsyncRetrying, ClientConnectionError)
```

Reply:
It will reduce duplication with sync backend